WinUI3/AK.Toolkit.WinUI3.Localization/Extensions.cs

```diff
@@ -9,6 +9,34 @@ namespace AK.Toolkit.WinUI3.Localization;
 
 public static class Extensions
 {
+    /// <summary>
+    /// You need to Initialize Window with 2 parameters
+    /// </summary>
+    /// <param name="Localizer"></param>
+    /// <param name="Root">Grid/StackPanel or any FrameworkElement that hosts elements</param>
+    /// <param name="Content">Windows `Content` Properties</param>
+    public static void InitializeWindowEx(this Localizer Localizer, FrameworkElement Root, UIElement Content)
```

Comment:
Do we need to pass the ``UIElement``? The localizer is supposed to find its children. Did you find any problems around there?

Reply:
Yes we need to pass Window.Content property.
No until here, i didnt see any issues.
in this case everything works fine if we put our views in mainwindow

```
<Window>
    <Grid x:Name="Root">
        <Grid.ColumnDefinitions>
            <ColumnDefinition/>
            <ColumnDefinition/>
        </Grid.ColumnDefinitions>
        <local:BlankPage1 Grid.Column="1"/>
        <Button x:Name="myButton" l:Localizer.Uid="String1" Click="myButton_Click"/>
        <Button x:Name="myButton2" Margin="200" l:Localizer.Uid="String1" Click="myButton_Click_1"/>
    </Grid>
</Window>
```

Reply:
Sorry but I can't think a case where ``Root`` and "Content" won't be the same in a ``Window``.🤔I also found that you are calling ``RunLocalization(Root)`` inside this method. Do you mean ``RegisterRootElement(Root)``?

BTW, After merging your pull request, I'll be make changes around the 2 steps (set language then run localization) and make it just 1 step (set language)🤔

Reply:
hmmm.... in your sample and MainWindow you are calling 
```
public MainWindow()
    {
        InitializeComponent();
        if (Content is FrameworkElement content)
        {
            _localizer.RegisterRootElement(content);
        }
    }
```
and in every pages you are calling

`_localizer.RunLocalization(Root);`

when i remove **RunLocalization(Root)**, The translation does not work for the first time

![Animation23](https://user-images.githubusercontent.com/9213496/187374346-031da462-d426-4cb2-8040-210b0b1a862b.gif)

and if i remove **_localizer.RegisterRootElement(content);** then dynamic changing language is not work.
![Animation234](https://user-images.githubusercontent.com/9213496/187374712-05c0984a-fc4b-49d4-acb4-668bb5732d4f.gif)

so we need both of them

> BTW, After merging your pull request, I'll be make changes around the 2 steps (set language then run localization) and make it just 1 step (set language)🤔
this will be good and we dont need **SetCurrentLanguageEx** and **SetCurrentLanguage**


for now this sample working fine...
![Animation2](https://user-images.githubusercontent.com/9213496/187373843-7fdd730b-7477-4d1a-8839-645e78792bd8.gif)

Reply:
I'm sorry. You are right. You need to call ``RunLocalization()`` in order to localize the Windows's content.🙏